cccs-build/superset/Dockerfile

```
@@ -1,7 +1,7 @@
 # Vault CA container import
 ARG VAULT_CA_CONTAINER=uchimera.azurecr.io/cccs/hogwarts/vault-ca:master_11376_a25c34e1
 FROM $VAULT_CA_CONTAINER AS vault_ca
-FROM uchimera.azurecr.io/cccs/superset-base:cccs-2.1_20230726145242_b7395
+FROM uchimera.azurecr.io/cccs/superset-base:CLDN-2271-disable-populating-the-value-filter-option_20230829194848_b7564
```

```suggestion
FROM uchimera.azurecr.io/cccs/superset-base:cccs-2.1_20230726145242_b7395
```
We shouldn't update this until the PR has been merged in